with-go/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"html/template"
+	"log"
+	"net/http"
+
+	"github.com/machinebox/graphql"
+)
+
+func main() {
+	http.HandleFunc("/", getProducts)
+	http.ListenAndServe(":8080", nil)
+}
+
+func getProducts(w http.ResponseWriter, r *http.Request) {
+	client := graphql.NewClient("https://api-eu-central-1.graphcms.com/v2/ck8sn5tnf01gc01z89dbc7s0o/master")
```

Comment:
Potentially you could move the client creation into the main function, but that's not required

Reply:
This would be useful actually. I want to introduce another route for /products/:slug